conda/recipes/rmm/recipe.yaml

```diff
@@ -6,7 +6,7 @@ context:
   cuda_version: ${{ (env.get("RAPIDS_CUDA_VERSION") | split("."))[:2] | join(".") }}
   cuda_major: ${{ (env.get("RAPIDS_CUDA_VERSION") | split("."))[0] }}
   date_string: ${{ env.get("RAPIDS_DATE_STRING") }}
-  py_version: ${{ env.get("RAPIDS_PY_VERSION") }}
+  py_version: ${{ env.get("RAPIDS_PY_VERSION") | version_to_buildstring }}
```

Comment:
> _`${{ python | version_to_buildstring }}` converts a version from the variant to a build string (it removes the . character and takes only the first two elements of the version)._

ref: https://rattler.build/v0.35.6/reference/jinja/#the-version_to_buildstring-function

Nice!